packages/react-composites/src/composites/MeetingComposite/ChatButton.tsx

```diff
@@ -4,24 +4,38 @@
 import React from 'react';
 import { ControlBarButton, ControlBarButtonProps } from '@internal/react-components';
 import { Chat20Filled, Chat20Regular } from '@fluentui/react-icons';
+import { NotificationIcon } from './NotificationIcon';
+import { Stack } from '@fluentui/react';
+
+/**
+ * @private
+ */
+export interface ChatButtonProps extends ControlBarButtonProps {
+  numberOfUnreadMessages?: number;
```

Comment:
what are our thoughts on unreadMessageCount

Its shorter and its clearer based on the type it represents?

Reply:
+1